Extensions/3D/Base3DBehavior.ts

```diff
@@ -103,6 +103,27 @@ namespace gdjs {
     flipZ(enable: boolean): void;
 
     isFlippedZ(): boolean;
+
+    /**
+     * Return the bottom Z of the object.
+     * Rotations around X and Y are not taken into account.
+     */
+    getAABBMinZ(): number;
+
+    /**
+     * Return the top Z of the object.
+     * Rotations around X and Y are not taken into account.
+     */
+    getAABBMaxZ(): number;
+  }
+
+  export namespace Base3DHandler {
+    export const is3D = (
+      o: gdjs.RuntimeObject
```

Comment:
Dont' use single name variable, they are a pain to multi select to refactor and impossible to grep.

Reply:
I don't like them either, I copied it from the tween extension and didn't notice.